internal/setup/setup.go

```diff
@@ -44,6 +45,10 @@ type Package struct {
 	Slices  map[string]*Slice
 }
 
+func (p *Package) MarshalYAML() (interface{}, error) {
+	return packageToYAML(p)
```

Comment:
I know we talked about this being used somewhere else in the past (see resolved conversation), but I tested it locally and we can inline `packageToYAML`, `sliceToYAML` and `pathInfoToYAML`. In fact, `Slice` does not need to implement `MarhsalYaml` so let's remove it.

Reply:
I am not quite sure what you mean here. I am removing the `Slice.MarshalYAML()` function.
Did you mean to get rid of the `Package.MarshalYAML` function too?